Bottom: 4ea9c531bb71f49dfa3a2778fe4ab49d5dd3df1f
Top:    b4d720389515a88af15141cd36bf63a1611e5789
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-12 16:14:27 +0000

Enable -Wextra (NOT yet -pedantic)


---

diff --git a/configure.ac b/configure.ac
index 058fa40..10d84b5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1143,7 +1143,7 @@ LIBS="$LIBS $XM_LIBS $DBUS_LIBS $X_LIBS $GLIB_LIBS $GTK_LIBS $DMALLOC_LIBS $GD_L
 # if we have gcc then add -Wall
 if test "x$GCC" = "xyes"; then
 	# see about adding some extra checks if the compiler takes them
-	for flag in -Wall ; do
+	for flag in -Wall -Wextra; do #-pedantic
 		case " ${CFLAGS} " in
 			*\ ${flag}\ *)
 				# flag is already present
